src/main/java/bio/terra/cli/businessobject/User.java

```diff
@@ -119,20 +133,35 @@ public void loadExistingCredentials() {
     }
   }
 
+  /**
+   * Load any existing credentials for this user. Prompt for browser login if they are expired or do
+   * not exist.
+   */
+  public static void login() {
+    login(/*useApplicationDefaultCredentials=*/ false);
+  }
+
   /**
    * Load any existing credentials for this user. Prompt for login if they are expired or do not
    * exist.
+   *
+   * @param useApplicationDefaultCredentials when true, use the application default credentials to
+   *     log-in instead of oauth flow in the browser.
    */
-  public static void login() {
+  public static void login(boolean useApplicationDefaultCredentials) {
```

Comment:
Maybe make this parameter an enum like `LoginMode` so it's more obvious at the call site and we can support more modes later.